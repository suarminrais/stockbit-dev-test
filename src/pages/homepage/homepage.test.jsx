import {
  shallow
} from "enzyme";
import React from "react";
import {
  Provider
} from "react-redux";
import configureStore from 'redux-mock-store'

import HomePage from './homepage.component'
const mockStore = configureStore();
const store = mockStore({});

it('expect to render HomePage Component', () => {
  expect(shallow(<Provider store={store}>
    <HomePage />
  </Provider>)).toMatchSnapshot()
})