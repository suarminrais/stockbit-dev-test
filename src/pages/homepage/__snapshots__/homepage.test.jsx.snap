// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render HomePage Component 1`] = `ShallowWrapper {}`;
